frontend/src/components/details/CanvasComponent.jsx

```diff
@@ -8,35 +8,44 @@ such as diagrams and tables
 import React, { Component } from 'react';
 import PieChart from './PieChartComponent';
 import ColumnChart from './ColumnChartComponent';
-import Table from './TableComponent'
-import './canvas.css';
-import {Container, Row, Col } from 'react-grid-system';
-
+import Table from './TableComponent';
+import { Container, Row, Col } from 'react-grid-system';
+import CompareCanvas from './CompareCanvas';
 
```

Comment:
Please add a documentation comment here.
```suggestion
/**
* The CanvasComponent displays either the Detailed Information and graphs for one model 
* of a specific product or a comparison between two models if requested.
*/
```

Reply:
> Please add a documentation comment here.

sure, thanks for your attention.

Reply:
ah I actually would like to have it as a header of the actual Component, rather than the file. That way when you hover over a used component the IDE will display you the documentation 😃 

So in line 14 would be better.

Reply:
![image](https://user-images.githubusercontent.com/22003632/118258621-68c64480-b4b0-11eb-8cab-4108d3169102.png)
 compared to 
![image](https://user-images.githubusercontent.com/22003632/118258683-7e3b6e80-b4b0-11eb-954c-5e04ccf7cb3d.png)

This is the different in the tooltip information that I'm talking about :)

Reply:
perfect, I didn't know that, I will refactor them. thanks :)